app/helpers/application_helper.rb

```diff
@@ -167,6 +167,17 @@ def client_aliases(client)
     end
   end
 
+  def patient_to_add_to_aliases(client)
```

Comment:
I think this can be simplified quite a bit with the change below for `add_patient_to_aliases`  Then let's just call it `client_health_aliases`